lean4/src/putnam_1971_a3.lean

```diff
@@ -9,7 +9,7 @@ theorem putnam_1971_a3
 (habclattice : a.1 = round a.1 ∧ a.2 = round a.2 ∧ b.1 = round b.1 ∧ b.2 = round b.2 ∧ c.1 = round c.1 ∧ c.2 = round c.2)
 (habcneq : a ≠ b ∧ a ≠ c ∧ b ≠ c)
 (hR : R > 0)
-(oncircle : (ℝ × ℝ) → ℝ → (ℝ × ℝ) → Prop := fun C r p => Euclidean.dist p C = r)
+(oncircle : (ℝ × ℝ) → ℝ → (ℝ × ℝ) → Prop := fun C r p => Real.sqrt ((p.1 - C.1)^2 + (p.2 - C.2)^2) = r)
 (hcircle : ∃ C : ℝ × ℝ, oncircle C R a ∧ oncircle C R b ∧ oncircle C R c)
```

Comment:
If you use `WithLP 2 (ℝ × ℝ)` here and elsewhere instead of `ℝ × ℝ`, then `dist` has the right meaning.

Reply:
Thanks! For this one, I think I'll change some problems which I come across and leave others as is. Some variation here can help identify weaknesses in methods people end up trying down the line. Statement should be correct either way but different methods might be more capable on one than the other, which should be useful info to have.